### PR TITLE
agent: Fix an issue reporting OOM events by mistake

### PR DESCRIPTION
**Pull request by justxuewei**
The agent registers an event fd in `memory.oom_control`. An OOM event is
forwarded to containerd when the event is emitted, regardless of the
content in that file.

I observed content indicating that events should not be forwarded, as shown
below. When `oom_kill` is set to 0, it means no OOM has occurred. Therefore,
it is important to check the content to avoid mistakenly forwarding OOM
events.

```
oom_kill_disable 0
under_oom 0
oom_kill 0
```

Fixes: #8715

### CODE REVIEW

File: src/agent/rustjail/src/cgroups/notifier.rs [hunk 1]
```diff
@@ -165,22 +166,17 @@ async fn register_memory_event(
 
     tokio::spawn(async move {
         loop {
-            let sender = sender.clone();
```
Comment (Apokleos) [inline src/agent/rustjail/src/cgroups/notifier.rs:168]: why the line deleted ?

Comment (justxuewei) [reply src/agent/rustjail/src/cgroups/notifier.rs:168]: Cuz `clone()` is unnecessary here.

**Review by Apokleos — APPROVED** [review]
Thx @justxuewei

**Review by Tim-Zhang — APPROVED** [review]

**Pull request merged**